packages/providers/mgt-msal2-provider/src/Msal2Provider.ts

```diff
@@ -107,6 +115,18 @@ export interface Msal2Config {
   options?: Configuration;
 }
 
+/**
+ * Prompt type enum
+ *
+ * @export
+ * @enum {number}
+ */
+export enum PromptType {
+  SELECT_ACCOUNT,
+  LOGIN,
+  CONSENT
```

Comment:
You could also include the strings here and avoid having to map each one with `getPromptType`

```suggestion
  SELECT_ACCOUNT = 'select_account,
  LOGIN = 'login',
  CONSENT = 'consent`
```

Alternatively, you don't need to use an enum and just use a type. ex:

```ts
export type PromptType = 'select_account' | 'login' | 'consent';
```

Either option will work, just a mater of preference.